lib/cocoapods-downloader/remote_file.rb

```diff
@@ -123,6 +123,8 @@ def extract_with_type(full_filename, type = :zip)
             FileUtils.move(entry.children, target_path)
           end
         end
+
+        FileUtils.rm(unpack_from) if File.exist?(unpack_from)
```

Comment:
the `scp_spec` circumvents the downloading causing the `file.zip` not to exist and make the tests fail.

Added the `if` statement to fix it instead, it should be harmless.